docs/design-proposals/global-scheduler/GlobalSchedulerArchitectureDesignSpecificationFirstDraft.md

```diff
@@ -0,0 +1,416 @@
+- [1. Introduction](#1-introduction)
+  - [1.1 Problem Statement](#11-problem-statement)
+  - [1.2 Goals and Scope](#12-goals-and-scope)
```

Comment:
Our design constraints 1K DCs+Edge sites ,the number of site depend on how we define the edge ： edge DC or  edge device。